app/controllers/UnloadingGuidanceController.scala

```diff
@@ -36,25 +35,18 @@ class UnloadingGuidanceController @Inject() (
   requireData: DataRequiredAction,
   navigator: Navigator,
   val controllerComponents: MessagesControllerComponents,
-  renderer: Renderer,
+  view: UnloadingGuidanceView,
   checkArrivalStatus: CheckArrivalStatusProvider
 )(implicit ec: ExecutionContext)
     extends FrontendBaseController
     with I18nSupport {
 
   def onPageLoad(arrivalId: ArrivalId, mode: Mode): Action[AnyContent] =
-    (identify andThen checkArrivalStatus(arrivalId) andThen getData(arrivalId) andThen requireData).async {
+    (identify andThen checkArrivalStatus(arrivalId) andThen getData(arrivalId) andThen requireData) {
```

Comment:
It might be worth Switching to use the combined Action that we're using in Arrivals. i.e. include the class Actions from Arrivals switching to require an arrivalId, and change this line to just be actions.requireData(arrivalId)